From: Anthony G. Basile <blueness@gentoo.org>
Updated patch for 2.6.38.6

The credits/description from the original version of this patch remain accurate
and are included below.

---
From: Jory A. Pratt <anarchy@gentoo.org>
Updated patch for kernel 2.6.32

The credits/description from the original version of this patch remain accurate
and are included below.

---
From: Gordon Malm <gengor@gentoo.org>

Updated patch for kernel series 2.6.24.

The credits/description from the original version of this patch remain accurate
and are included below.

---
From: Alexander Gabert <gaberta@fh-trier.de>

This patch removes the warnings introduced by grsec patch 2.1.9 and later.
It removes the -W options added by the patch and restores the original
warning flags of vanilla kernel versions.

Acked-by: Christian Heim <phreak@gentoo.org>
---

--- a/Makefile	2011-05-20 08:12:41.000000000 -0400
+++ b/Makefile	2011-05-20 08:18:18.000000000 -0400
@@ -233,7 +233,7 @@
 
 HOSTCC       = gcc
 HOSTCXX      = g++
-HOSTCFLAGS   = -Wall -W -Wmissing-prototypes -Wstrict-prototypes -Wno-empty-body -Wno-unused-parameter -Wno-missing-field-initializers -O2 -fomit-frame-pointer -fno-delete-null-pointer-checks
+HOSTCFLAGS   = -Wall -Wmissing-prototypes -Wstrict-prototypes -Wno-empty-body -Wno-unused-parameter -Wno-missing-field-initializers -O2 -fomit-frame-pointer -fno-delete-null-pointer-checks
 HOSTCXXFLAGS = -O2 -fno-delete-null-pointer-checks
 
 # Decide whether to build built-in, modular, or both.
